src/zope/browserresource/tests/test_i18nfile.py

```diff
@@ -37,10 +37,33 @@
 from zope.i18n.interfaces import INegotiator
 from zope.i18n.negotiator import negotiator
 
-from zope.i18n.tests.testii18naware import TestII18nAware
 
 test_directory = os.path.dirname(p.__file__)
 
+class AbstractTestII18nAwareMixin(object):
+
+
+    def setUp(self):
+        super(AbstractTestII18nAwareMixin, self).setUp()
+        self.object = self._createObject()
+        self.object.setDefaultLanguage('fr')
+
+    def _createObject(self):
+        # Should create an object that has lt, en and fr as available
+        # languages
+        raise NotImplementedError()
+
+    def testGetDefaultLanguage(self):
+        self.assertEqual(self.object.getDefaultLanguage(), 'fr')
+
+    def testSetDefaultLanguage(self):
+        self.object.setDefaultLanguage('lt')
+        self.assertEqual(self.object.getDefaultLanguage(), 'lt')
+
+    def testGetAvailableLanguages(self):
+        self.assertEqual(sorted(self.object.getAvailableLanguages()), ['en', 'fr', 'lt'])
+
+
 
 @adapter(IFileResource, IBrowserRequest)
```

Comment:
One too many blank lines here also (should be two, not three, between top-level classes).

(Unless you did it to be consistent with the rest of the file, which is commendable.)